src/test/java/org/minions/devfund/mauricioz/linkedList/InsertSortedTest.java

```diff
@@ -0,0 +1,181 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+// Source code recreated from a .class file by IntelliJ IDEA.
+// (powered by Fernflower decompiler)
+*/
+public class InsertSortedTest {
+    private static final int LISTFIRSTVALUE = 3;
+    private static final int LISTSECONDVALUE = 8;
+    private static final int LISTTHIRDVALUE = 11;
+
+    private static final int LISTSIZE1 = 4;
+    private static final int LISTSIZE2 = 3;
+
+    private static final int LISTNEWVALUE1 = 10;
+    private static final int LISTNEWVALUE2 = 14;
+    private static final int LISTNEWVALUE3 = 1;
+
+    private static final int LISTPOSITION3 = 3;
+    /**
+     * constructor.
+     */
+    public InsertSortedTest() {
+    }
+
+    /**
+     * setup for test.
+     */
+    @Before
+    public void setUp() {
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertMiddle() {
+        LinkedList var1 = new LinkedList();
+        var1.add(LISTFIRSTVALUE);
+        var1.addLast(LISTSECONDVALUE);
+        var1.addLast(LISTTHIRDVALUE);
+
+        try {
+            LinkedListUtils.insertSorted(var1, LISTNEWVALUE1);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value in middle of list");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "insertSorted throws " 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)